community/modules/scripts/spack-install/outputs.tf

```diff
@@ -14,33 +14,61 @@
  * limitations under the License.
  */
 
-output "startup_script" {
```

Comment:
(No action needed)

I see what you meant by this. A future solution may be to create a `startup-script` module as part of this module and provide it's output here, but that's a low priority for now.